store/views.py

```diff
@@ -25,11 +25,22 @@ def bookDetailView(request, bid):
 @csrf_exempt
 def bookListView(request):
     template_name = 'store/book_list.html'
+    get_data = request.GET
+    
+    try:
+        books = Book.objects.filter(
+            title__contains=get_data['title'],
+            author__contains=get_data['author'],
+            genre__contains=get_data['genre'],
```

Comment:
You could have used `icontains` here for a case-insensitive match. However, this is fine for this assignment.